Trustroots/trustroots
# Build: stop auto-detecting outdated dependencies

#### Proposed Changes

Modify the script that detects outdated `node_modules` and runs install. It now just warns when dependencies are missing or outdated.

![image](https://user-images.githubusercontent.com/87168/102901462-56525c80-4476-11eb-82d4-17eb24e0fbb3.png)

![image](https://user-images.githubusercontent.com/87168/102901567-7550ee80-4476-11eb-825e-4873c4c6ab40.png)

I added this to avoid confusion new and inexperienced volunteers were having with installing and keeping dependencies up to date in their setup.

Still a fair concern, but I'd rather deal with educating folks into NPM procedures than deal with getting dependencies installed each time I want to review someone's PR. :-)

We just need to be mindful and run `npm ci` to refresh dependencies ourselves when needed.

#### Testing Instructions

* Run `touch package.json`
* Run `npm start`; no install should occur
